dockerfiles/run/guice/cassandra-rabbitmq/destination/conf/mailetcontainer.xml

```diff
@@ -78,6 +78,11 @@
                 <notice>550 - Requested action not taken: no such user here</notice>
             </mailet>
 
+	    <mailet match="SenderIsLocal" class="ToSenderFolder">
+	    	<folder>Sent</folder>
+		<consume>false</consume>
+	    </mailet>
```

Comment:
Generally the mua takes care of saving stuff in sent folder.

While I agree about improving the doc, I am unsure this change should make it to default configuration...

Reply:
[Generally the mua takes care of saving stuff in sent folder.](url)
Think about the gmail/netease, the mail server will save the sent letter to the mail server by default. In my opinion, james should work save as well, If they want to turn it off, they can remove the ToSentFolder, not the other way.

Reply:
I have thunderbird configured on top of gmail, when I send a mail via smtp, there is a second imap request to save the mail in sent folder.

While better documenting how "to sender foler wotks" i really doubt enabling it by default would be a good move.

Cheers,

Benoit

Reply:
> Think about the gmail/netease, the mail server will save the sent letter to the mail server by default. In my opinion, james should work save as well, If they want to turn it off, they can remove the ToSentFolder, not the other way.

I some users want to configure this, it's more convenient to let the configuration of this at the MUA side than on the server side.